### ISSUE TITLE
_cpyfrom error on date time field

### ISSUE THREAD

**Issue by topdog-BigDogWerks** [cortexrd/Knack-Toolkit-Library#386]
Using _cpyfrom to create data in destination grid resulted in some errors. 

Furethr testing.
I created two new Tables/Objects
_test and _test 2
_test 2 is a copy of _test so the all the fields are teh same/same type

I created 1 record in _test and clicked the “copy” Button.
Here is the result.
![image](https://github.com/user-attachments/assets/dc5e6f24-84d5-4d74-a91f-14115cda2462)

See Knack Forum post https://forums.knack.com/t/introducing-the-knack-toolkit-library-ktl-in-pre-relase-by-cortex-r-d-inc/11046/84

**Comment by cortexrd** [comment]
Erik,

I've fixed your issues.  Can you test the **Dev** code please?
Also try the keyword in other places if you can.  There are so many things that can go wrong with this one, it's very complex!

Norm

**Comment by topdog-BigDogWerks** [comment]
What do I change in my JS to run DEV code?
You can point me to the documentation. 
In the 
mean time, I'll go looking there.

**Comment by cortexrd** [comment]
Switching to Dev and Beta is explained here:  https://github.com/cortexrd/Knack-Toolkit-Library/discussions/186

**Comment by topdog-BigDogWerks** [comment]
Got it. 
Definitely works for that test view. I will test in the planned view as well.

**Comment by topdog-BigDogWerks** [comment]
I probably should have closed this Issue, but now I am back testing this keyword. 
But, now that I am testing the function again, I'm having a different issue. 
**Let me know if I should close this and open a new issue.**

I have the keyword making copies as expected. But it is not copying a Number field from Hours to Hours.

![Image](https://github.com/user-attachments/assets/5653f8b1-b80b-4cd8-8714-82bb56fea064)

**Comment by cortexrd** [comment]
No, let's keep adding to this same issue.

Question:  am I right to say that the source is an Equation and the destination is a Number field?

**Comment by topdog-BigDogWerks** [comment]
> No, let's keep adding to this same issue.
> 
> Question: am I right to say that the source is an Equation and the destination is a Number field?

No, both are number fields.

**Comment by cortexrd** [comment]
Got it.  I'm in the Builder now, and saw that both are numbers.

Can I create some data to test?

**Comment by topdog-BigDogWerks** [comment]
> Got it. I'm in the Builder now, and saw that both are numbers.
> 
> Can I create some data to test?

Absolutely.
This is an application in development and you can create whatever you like.

**Comment by cortexrd** [comment]
Fixed and released in **KTL v0.29.8**

**Comment by topdog-BigDogWerks** [comment]
Seems to function as documented. 
Should this keyword refresh the destination table when the function is complete?

**Comment by cortexrd** [comment]
No, the Add option only works when the destination is empty.  To update you must first wipe out all records.
